protocol/v2/ssv/runner/runner.go

```
@@ -309,39 +320,51 @@ func (b *BaseRunner) baseConsensusMsgProcessing(ctx context.Context, logger *zap
 		return false, nil, fmt.Errorf("invalid post-consensus message: %w", err)
 	}
 
+	vIndices := make([]uint64, 0, len(signedMsg.Messages))
+	for _, msg := range signedMsg.Messages {
+		vIndices = append(vIndices, uint64(msg.ValidatorIndex))
+	}
 	const gotPostConsensusMsgEvent = "📬 got post-consensus message"
-	logger.Debug(gotPostConsensusMsgEvent, zap.Uint64("signer", ssvtypes.PartialSigMsgSigner(signedMsg)))
+	logger.Debug(
+		gotPostConsensusMsgEvent,
+		zap.Uint64("signer", ssvtypes.PartialSigMsgSigner(signedMsg)),
+		zap.Uint64s("validators", vIndices),
+	)
 	span.AddEvent(gotPostConsensusMsgEvent)
 
-	hasQuorum, roots := b.basePartialSigMsgProcessing(signedMsg, b.State.PostConsensusContainer)
+	hasQuorum, quorumRoots := b.basePartialSigMsgProcessing(signedMsg, b.State.PostConsensusContainer)
 
 	if hasQuorum {
 		const gotPostConsensusQuorumEvent = "🎯 got post-consensus quorum"
-		logger.Debug(gotPostConsensusQuorumEvent)
+		logger.Debug(gotPostConsensusQuorumEvent, fields.QuorumRoots(quorumRoots))
 		span.AddEvent(gotPostConsensusQuorumEvent)
 	}
 
-	return hasQuorum, roots, nil
+	return hasQuorum, slices.Collect(maps.Keys(quorumRoots)), nil
 }
 
 // basePartialSigMsgProcessing adds a validated (without signature verification) validated partial msg to the container, checks for quorum and returns true (and roots) if quorum exists
 func (b *BaseRunner) basePartialSigMsgProcessing(
 	signedMsg *spectypes.PartialSignatureMessages,
 	container *ssv.PartialSigContainer,
-) (bool, [][32]byte) {
-	roots := make([][32]byte, 0)
-	quorumReached := false
+) (gotAnyQuorum bool, roots map[[32]byte]map[phase0.ValidatorIndex][]spectypes.OperatorID) {
+	roots = make(map[[32]byte]map[phase0.ValidatorIndex][]spectypes.OperatorID)
 
 	for _, msg := range signedMsg.Messages {
-		quorumReachedPreviously := container.HasQuorum(msg.ValidatorIndex, msg.SigningRoot)
+		quorumReachedPreviously, _ := container.HasQuorum(msg.ValidatorIndex, msg.SigningRoot)
```

is that an ignored error in the `_`?

---

i think not, but just checking... ignored last values make me uneasy.

---

It's not an error that `container.HasQuorum` returns but a "list of signers/operators" ... we just don't need these signers from this specific call here (in other places these signers/operators are used to print detailed debug log-lines)